### PR TITLE
adding cursors

### PR DESCRIPTION
**Pull request by eewayhsu**

### CODE REVIEW

File: assets/js/controllers/toolbar.js [hunk 1]
```diff
@@ -6,16 +6,27 @@ $('#ToolbarView').on('click', '.toolbar-btn', function(e) {
 	var toolName = tool.prop('id')
 	if (toolName === "select-btn") {
 		easel.currentTool = tacit.tools.select
+		$("#PadView").css({'cursor': 'default'})
 	} else if (toolName === "move-btn") {
 		easel.currentTool = tacit.tools.move
+		$("#PadView").css({'cursor': 'pointer'})
 	} else if (toolName === "draw-btn") {
 		easel.currentTool = tacit.tools.draw
+		$("#PadView").css({'cursor': 'url(assets/resources/cursor-images/pencil.png) 0 22, auto'})
```
Comment (bqpd) [inline assets/js/controllers/toolbar.js]: What's up with the 0, 22, auto?

**Comment by eewayhsu** [comment]
Sorry magic numbers.  It's the x y position of the cursor hotspot. 
so its at the bottom left corner.  I didnt think of any better ways

**Comment by eewayhsu** [comment]
Also this hard-coded box is there for the padview for GR5.  I didn't look through how the space was getting set, but it does some strange movement with regards to the easel.  This has to be fixed.

**Comment by bqpd** [comment]
Sorry, what's the box view do? (I should just pull and play with it, shouldn't I).

**Comment by bqpd** [comment]
Ah, so it makes the border?

Cursor magic numbers don't work on my computer. Let's just flip the pencil.

**Comment by eewayhsu** [comment]
Merged.

**Comment by bqpd** [comment]
Did you push the merged master? Because mine doesn't have it...you may have to re-merge :p

**Comment by mmou** [comment]
i'm also not seeing this in master

**Pull request closed without merging**